dali_tf_plugin/dummy/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required(VERSION 3.2)
+project(dali_tf_dummy)
+
+set(DALI_ROOT "${PROJECT_SOURCE_DIR}/../..")
```

Comment:
That's baaad. It's rather against the idea of the project, which shall not need reference "above".

Why not just use `add_subdirectory` for this?

Reply:
this is a totally separated project. It's not part of the DALI cmake. Something on the side that can be built on its own. The reason we need DALI_ROOT is because we want to access to the cmake dir with the common utilities

Reply:
Since it needs to have an access to `cmake/Utils.cmake`, we can't build it on its own. If I copy `dali_tf_plugin` out of DALI repo, I can't build it anymore...

Reply:
It doesn't need DALI to be built, it just needs `Utils.cmake`. This is not part of the main DALI project and it's not built in the same step. Rather than copying `Utils.cmake` once again, we just have a reference to the top dir of the repository. Do you have a better suggestion?

Reply:
OK, let it be.
But I think you shouldn't create new variable, but just use the one that's already there: `${DALI_SOURCE_DIR}`. That's variable generated by `project` directive in CMake.
like:
```
include (${DALI_SOURCE_DIR}/cmake/Utils.cmake)
```

Reply:
As we discussed offline, that's not possible because this is not a subproject of the main DALI project. It is something completely separate